src/datepicker/datepicker-service.ts

```diff
@@ -151,6 +151,15 @@ export class NgbDatepickerService {
     return this._calendar.isValid(ngbDate) ? ngbDate : defaultValue;
   }
 
+  getMonth(struct: NgbDateStruct) {
+    for (let viewModel of this._state.months) {
```

Comment:
nit: maybe `month` instead of `viewModel`?

Reply:
It is the input of NgbDatepickerMonthView. User might provide a an object with just the year and the month and not a proper NgbDate object.

Reply:
it was a nitpick and I was talking purely about the `viewModel` variable name